cluster/common.sh

```diff
@@ -99,7 +109,10 @@ function create-kubeconfig() {
     "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
   fi
   "${kubectl}" config set-context "${CONTEXT}" --cluster="${CONTEXT}" --user="${CONTEXT}"
-  "${kubectl}" config use-context "${CONTEXT}"  --cluster="${CONTEXT}"
+
+  if [[ "${SECONDARY_KUBECONFIG}" != "" ]];then
```

Comment:
That's correct, my mistake. I'm going to change it to ` != "true"`. To bypass setting this context as current, set `SECONDARY_KUBECONFIG=true`.